### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4767
Added information to the NOTE section.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -81,7 +81,8 @@ You have several ways to set the Operator:
 - **Person in your organization** assigns a person with a Phone System license that is enabled for Enterprise Voice or assigned Calling Plans in Office 365. You can also set it up so the caller is sent to voicemail. To send a caller to voicemail, select **Person in your organization** and set that account's settings to send calls directly to voicemail.
 
      > [!Note]
-     > **Person in your organization** can be an Online user or a user hosted on-premises using Skype for Business Server.
+     > **Person in your organization** can be an Online user or a user hosted on-premises using Skype for Business Server. 
+     When selecting 'Person in Organization' you can select an account with a shared mailbox or with a user mailbox.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:85]:
```suggestion
     When selecting **Person in your organization** you can select an account with a shared mailbox or with a user mailbox.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 2]
```diff
@@ -81,7 +81,8 @@ You have several ways to set the Operator:
 - **Person in your organization** assigns a person with a Phone System license that is enabled for Enterprise Voice or assigned Calling Plans in Office 365. You can also set it up so the caller is sent to voicemail. To send a caller to voicemail, select **Person in your organization** and set that account's settings to send calls directly to voicemail.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:81]:
```suggestion
- **Person in organization** assigns a person with a Phone System license that is enabled for Enterprise Voice or assigned Calling Plans in Office 365. You can also set it up so the caller is sent to voicemail. To send a caller to voicemail, select **Person in organization** and set that account's settings to send calls directly to voicemail.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 3]
```diff
@@ -81,7 +81,8 @@ You have several ways to set the Operator:
...
 
      > [!Note]
-     > **Person in your organization** can be an Online user or a user hosted on-premises using Skype for Business Server.
+     > **Person in your organization** can be an Online user or a user hosted on-premises using Skype for Business Server.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:84]:
```suggestion
     > **Person in organization** can be an Online user or a user hosted on-premises using Skype for Business Server. 
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 4]
```diff
@@ -158,7 +159,8 @@ If you select **Disconnect**, the caller is disconnected after the greeting play
   - **Person in organization** The account you choose must have a Phone System license enabled for Enterprise Voice or have an assigned Calling Plan in Office 365. You can set it up so the caller can be sent to voicemail: select **Person in organization** and set that account to have calls forwarded directly to voicemail.
 
   > [!Note]
-  > **Person in organization** can be an Online user or a user hosted on-premises using Skype for Business Server.
+  > **Person in organization** can be an Online user or a user hosted on-premises using Skype for Business Server. When selecting
+  'Person in Organization' you can select an account with a shared mailbox or with a user mailbox.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:163]:
```suggestion
  **Person in organization** you can select an account with a shared mailbox or with a user mailbox.
```

**Comment by scanum** [comment]
@kenwith PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**